src/nuclio/functions/version/version-code/function-event-pane/function-event-pane.component.js

```diff
@@ -232,6 +232,18 @@
             updateRequestHeaders();
         }
 
+        /**
+         * Changes method
+         * @param {Object} changes
+         */
+        function onChanges(changes) {
+            if (!changes.version.isFirstChange() &&
+                (lodash.has(changes.version.currentValue, 'status.externalInvocationUrls') ||
+                lodash.has(changes.version.currentValue, 'status.externalInvocationUrls'))) {
+                updateInvocationUrls()
+            }
```

Comment:
Missing a semicolon at the end of `updateInvocationUrls()`.

I suggest this code formatting for better readability:

```suggestion
            if (
                !changes.version.isFirstChange() &&
                (lodash.has(changes.version.currentValue, 'status.externalInvocationUrls') ||
                lodash.has(changes.version.currentValue, 'status.externalInvocationUrls'))
            ) {
                updateInvocationUrls()
            }
```
This way it is clearer where the `if` condition expression ends and the statement block starts.